clue.js

```diff
@@ -0,0 +1,282 @@
+// The Killers 
+function killers() {
+  const mrGreen = {
+    firstName: 'Jacob',
+    lastName: 'Green',
+    color: 'green',
+    description: 'He has a lot of connections',
+    age: 45,
+    image: 'media/green.png',
+    occupation: 'Entrepreneur'
+  }
+  const profPlum = {
+    firstName: 'Victor',
+    lastName: 'Plum',
+    color: 'purple',
+    description: 'Has the brains of Albert Einstein',
+    age: 65,
+    image: 'media/plum.png',
+    occupation: 'Scientist'
+  }
+  const msScarlet = {
+    firstName: 'Cassandra',
+    lastName: 'Scarlett',
+    color: 'red',
+    description: 'She will do anything for a profit',
+    age: 22,
+    image: 'media/scarlet.png',
+    occupation: 'Movie star'
+  }
+  const mrsPeacock = {
+    firstName: 'Eleanor',
+    lastName: 'Peacock',
+    color: 'blue',
+    description: 'maintains her dignity in almost all cases',
+    age: 32,
+    image: 'media/peacock.png',
+    occupation: 'Socialite'
+  }
+
+  const colMustard = {
+    firstName: 'Jack',
+    lastName: 'Mustard',
+    color: 'yellow',
+    description: 'Great white hunter and colonial imperialist',
+    age: 55,
+    image: 'media/mustard.png',
+    occupation: 'Colonel'
+  }
+  const mrsWhite = {
+    firstName: 'Mrs',
+    lastName: 'White',
+    color: 'lightgray',
+    description: 'She used to practice medicine and knows how to clean up blood',
+    age: 60,
+    image: 'media/white.png',
+    occupation: 'Housekeeper'
+  }
+
+  const suspects = [mrGreen, profPlum, msScarlet, mrsPeacock, colMustard, mrsWhite]
+  return suspects
```

Comment:
This can be done without a function actually! You just call `suspects` where you want to use it